projects/driver_controls/inc/can_output.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+// Module for broadcasting CAN messages
+#include <stdint.h>
+#include "status.h"
+
+// Device ID definitions
+typedef enum {
+  CAN_DEVICE_ID_POWER = 0,
+  CAN_DEVICE_ID_PEDAL,
+  CAN_DEVICE_ID_DIRECTION_SELECTOR,
+  CAN_DEVICE_ID_TURN_SIGNAL,
+  CAN_DEVICE_ID_HAZARD_LIGHT,
+  CAN_DEVICE_ID_MECHANICAL_BRAKE,
+  CAN_DEVICE_ID_HORN,
+  CAN_DEVICE_ID_PUSH_TO_TALK,
+} CANDeviceID;
```

Comment:
This should be `CANOutputDevice` or something similar and have `NUM_CAN_OUTPUT_DEVICES` < note the plurality

Reply:
Done